rero_ils/query.py

```diff
@@ -2,6 +2,7 @@
 #
 # RERO ILS
 # Copyright (C) 2019 RERO
+# Copyright (C) 2020 UCLOUVAIN
```

Comment:
It's `UCLouvain` ^^

Reply:
OK, I will fix that everywhere in a next PR